ocp_resources/virtual_machine_instance.py

```diff
@@ -117,16 +117,12 @@ def wait_until_running(self, timeout=TIMEOUT_4MINUTES, logs=True, stop_status=No
             TimeoutExpiredError: If VMI failed to run.
         """
         try:
-            self.logger.info(
```

Comment:
edit the PR title, this PR only remove some logs.
You wanted those logs, why do you remove them?

Reply:
title is still relevant as the pr is to fix flakiness caused by this log
also your pr https://github.com/RedHatQE/openshift-python-wrapper/pull/1367/ added more logging that makes this one redundant